package.json

```diff
@@ -39,6 +39,7 @@
     "babel-preset-env": "^1.6.1",
     "browser-sync": "^2.18.12",
     "browserstack-runner": "^0.7.0",
+    "chai": "^4.1.2",
```

Comment:
We must update both `package-lock.json` and `yarn.lock`. Please do the following:
```bash
rm package-lock.json
yarn
npm i --package-lock-only
sed -i '' 's/http\:\/\//https\:\/\//g' package-lock.json
sed -i '' 's/http\:\/\//https\:\/\//g' yarn.lock
```

Reply:
Hm, thought we use snyp or something similar. Will do it when I come to the installation of yarn.

Reply:
I took care of that.